src/main/java/frc/robot/subsystems/shooter/ShooterBedIOSparkMax.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.shooter;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.AnalogEncoder;
+import frc.robot.Constants;
+
+public class ShooterBedIOSparkMax implements ShooterBedIO
+{
+    private CANSparkMax   _bedLeaderSparkMax;
+    private CANSparkMax   _bedFollowerSparkMax;
+    private AnalogEncoder _bedAbsoluteEncoder;
+    private Rotation2d    _bedAbsoluteEncoderOffset;
+
+    @SuppressWarnings("resource")
+    public ShooterBedIOSparkMax(Rotation2d bedAbsoluteEncoderOffset)
```

Comment:
I forgot to mention this, but let's not pass the absolute offset in as a parameter to the constructor. Ideally, this will be managed by the dashboard.

Let's initialize `_bedAbsoluteEncoderOffset` to `Constants.BED_ANGLE_OFFSET` in the constructor directly, just like the CAN IDs